adagrams/game.py

```diff
@@ -1,11 +1,128 @@
+import random
 def draw_letters():
-    pass
+    letters = {
+            'A': 9, 
+            'B': 2, 
+            'C': 2, 
+            'D': 4, 
+            'E': 12,           
+            'F': 2, 
+            'G': 3, 
+            'H': 2, 
+            'I': 9, 
+            'J': 1, 
+            'K': 1, 
+            'L': 4, 
+            'M': 2, 
+            'N': 6,     
+            'O': 8, 
+            'P': 2, 
+            'Q': 1, 
+            'R': 6, 
+            'S': 4, 
+            'T': 6, 
+            'U': 4, 
+            'V': 2, 
+            'W': 2, 
+            'X': 1, 
+            'Y': 2, 
+            'Z': 1
+    }
+
+    drawn_letter = []
+    all_list = letters.items()
+    mynew_list = list(all_list)
+    
+    while len(drawn_letter) < 10:
+        random_number = random.randrange(0,len(mynew_list))
+        letter_tuples = mynew_list[random_number]
+        key = letter_tuples[0]
+        value = letter_tuples[1]
+        count_key_in_list = drawn_letter.count(key)
+
+        if count_key_in_list < value:
+            drawn_letter.append(key)
+
+    print(drawn_letter)
+    return drawn_letter
 
 def uses_available_letters(word, letter_bank):
-    pass
+    
+    found_count = 0
+    for i in letter_bank:
+        for j in word.upper():
```

Comment:
Great job Rika on implementing a nested loop! This shows me that you are showing proficiency in them! An alternative way you could have implemented this is: 
```python
def uses_available_letters(word, letter_bank):
    cap_word = word.upper()

    for letter in cap_word:
       if letter not in letter_bank: return False

       if cap_word.count(letter) > letter_bank.count(letter): return False
    
    return True
```
Python has a method for iterables called `count` and what it does is count the instances of the value passed to it inside of the iterable it is being called on. That way we don't have to create another loop to compare each `letter` in `word` to each element in `letter_bank`.